pom.xml

```diff
@@ -22,38 +22,37 @@
   <description> A statistics framework used inside Ehcache and the Terracotta products</description>
   <url>https://github.com/Terracotta-OSS/terracotta-management/</url>
 
+  <prerequisites>
```

Comment:
Since you go around fixing the build - please use the toolchain system to force it to be built with Java 6.